src/extension.ts

```diff
@@ -621,9 +634,9 @@ export async function activate(context: vscode.ExtensionContext) {
                                         break;
                                 }
                             } catch (err) {
-                                console.log(
-                                    `Non-JSON output from the process :  ${message}`
-                                );
+                                // if not a JSON string, it is incomplete
```

Comment:
Currently we assume that the error caught is the one we are looking for (Perhaps `SyntaxError` in this case?).
But that might not be always the case.
For e.g.: If a developer made a typo like:
```
messageToWebview = JSON.parse(mesage);
```
in the try block, we would put the message in the buffer even though it's a ReferenceError (`ReferenceError: mesage is not defined`).
You can check the error "type" like this to make sure we have the correct exception:
```
if (err.name === "SyntaxError") {
  ...
} else { // unhandled exception
  throw err;
}
```

Reply:
Changed!

Reply:
This is better than what I suggested, thanks for addressing!